container/pcct/README.md

```diff
@@ -3,41 +3,41 @@
+DPDK's hash library is unsuitable for this hash table because it requires fixed-length keys, but NDN names are variable length, and padding them to a fixed maximum length would be too wasteful.
 
-The `PccKey` type represents an index key of the name hash table,
-For a PIT entry, `PccKey` contains the Interest name and, if the Interest has a forwarding hint, the name of the chosen delegation.
-For a CS entry, `PccKey` contains the Data name and, if the Interest used to retrieve this Data has a forwarding hint, the name of the chosen delegation.
-Interest/Data name and delegation name are copied into `PccKey`, which has room for short and medium length names.
-If the names are too long to fit into `PccKey` itself, `PccKeyExt` objects can be allocated from the PCCT's mempool to provide extra room.
-Copying the names, rather than referencing them from the packet (usually an Interest), ensures the name buffers are NUMA-local, and allows the packet buffer holding the Interest to be freed when the Interest is satisfied.
+The `PccKey` type represents an index key of the name hash table.
+For a PIT entry, `PccKey` contains the Interest name and, if the Interest carries a forwarding hint, the name of the chosen delegation.
+For a CS entry, `PccKey` contains the Data name and, if the Interest used to retrieve this Data carried a forwarding hint, the name of the chosen delegation.
+Interest/Data name and delegation name are copied into the `PccKey`, which has room for short and medium-length names.
+If the names are too long to fit into the `PccKey` itself, `PccKeyExt` objects can be allocated from the PCCT's mempool to provide extra room.
+Copying the names, rather than referencing them from the packet (usually an Interest), ensures the name buffers are NUMA-local and allows the packet buffer holding the Interest to be freed when the Interest is satisfied.
 
-Searching the name hash table uses `PccSearch` type instead of `PccKey` type.
+Searching the name hash table uses the `PccSearch` type instead of the `PccKey` type.
 `PccSearch` contains pointers to linearized names.
-This avoids copying these names if they are not fragmented in the input packet.
-Searching with a different type is made possible by overriding the `uthash_memcmp` hook with `PccKey_MatchSearchKey` function.
+This avoids copying the names if they are not fragmented in the input packet.
+Searching with a different type is made possible by overriding the `uthash_memcmp` hook with the `PccKey_MatchSearchKey` function.
 
-Calling code must provide hash values.
-Hash value for an index key without forwarding hint is `Name_ComputeHash(name)`.
-Hash value for an index key with forwarding hint is `Name_ComputeHash(name) ^ Name_ComputeHash(delegation)`.
+Calling code must provide the hash values.
+The hash value for an index key without a forwarding hint is `Name_ComputeHash(name) FIXME`.
```

Comment:
This function no longer exists in the codebase

Reply:
https://github.com/usnistgov/ndn-dpdk/blob/4a12556b3b34c4f0de8a02f91a9c57e8be3fabb6/csrc/ndn/name.h#L24